intersphinx_registry/__init__.py

```diff
@@ -14,31 +14,37 @@
 
 registry_file = Path(__file__).parent / "registry.json"
 
+def _get_all_mappings() ->  Dict[str, Tuple[str, Optional[str]]]:
+    return cast(
+        Dict[str, Tuple[str, Optional[str]]],
+        {k: tuple(v) for (k, v) in json.loads(registry_file.read_bytes()).items()},
+    )
+
 
 def get_intersphinx_mapping(
-    *, only: Optional[Set[str]] = None
+    *, packages: Set[str] = set()
```

Comment:
Bad practice to use an empty container directly as a default, though it likely doesn't matter in actual usage because this function should(?) only get called once per Python session and thus there shouldn't(?) be opportunity for accidental reuse.

I forget exactly how to replace this with a suitable factory, though.

Reply:
Even if it's called many time, it is not muted in the function body and not returned to the user, so there is no risk of mutable default.